controllers/front/ListComments.php

```diff
@@ -65,7 +65,7 @@ public function display()
             $productComment['content'] = htmlentities($productComment['content']);
             $productComment['date_add'] = $dateFormatter->format($dateAdd);
 
-            if ($isLastNameAnynomus) {
+            if ($isLastNameAnynomus && isset($productComment['firstname']) && isset($productComment['lastname'])) {
```

Comment:
Hello

What's the idea behind checking if firstname is set if we don't use it in the condition ?

Reply:
Hi, 
`isset($productComment['firstname']) && isset($productComment['lastname'])` - for deleting account firstname and lastname is not set. I just want to be sure that nothing weird are going to happen when first name will not be available.

Reply:
Well, you are right. Check exists on line 74.